bioconda_utils/bioconda_utils-conda_build_config.yaml

```diff
@@ -48,3 +48,8 @@ numpy:
   - 1.16
   - 1.16
 
+channel_sources:
+  - conda-forge,bioconda,defaults
+
+channel_targets:
```

Comment:
Ouch... Yep, checked conda-build's source... This is now hard-coded to be included... I can't say I'm a big fan of including hashes for something which should be purely output-related :/.
(^-- Just a rant about conda-build's, not your changes.)
(My best guess they included it for their convenience since some recipes on conda-forge/AnacondaRecipes use `channel_targets` in some conditional Jinja control code so they don't have to manually sync the recipes.)